connection.impl/src/main/java/com/ericsson/bss/cassandra/ecchronos/connection/impl/builders/DistributedJmxBuilder.java

```diff
@@ -131,7 +131,8 @@ public final DistributedJmxConnectionProviderImpl build() throws IOException
         createConnections();
         return new DistributedJmxConnectionProviderImpl(
                 myNodesList,
-                myJMXConnections
+                myJMXConnections,
+                this
         );
```

Comment:
Not sure this is the correct way of doing this. 
I need code that is part of the builder so when nodes are added or deleted, or currently pass the builder in and re use it. 
But maybe this should more of a refactor to take the required code out of the builder